keyboards/handwired/dactyl_coolio/keymaps/id-b3/config.h

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Please change the `-` in the keymap folder name to `_`.

Reply:
The folder name is linked to my github username, and to the userspace. I see other users with - in their folder names and keymaps on the main repo.

Reply:
Okay, I will just note then that in some circumstances keyboard/keymap names with hyphens can cause problems, usually keyboard name as it is used in defines which cannot contain `-`. For example, `keyboardname/rev1` = `KEYBOARD_keyboardname_rev1`, often used to switch out code depending on the board being built.

But if it compiles, then no problem :)

Reply:
Thanks for the explanation! I haven't been having any issues with compiling locally so far! Thank you for the review, I'll update the rest of the changes and hopefully will be ready to finish the pull request!